src/test/java/api/loans/DeclareLostAPITests.java

```diff
@@ -110,13 +125,274 @@ public void cannotDeclareItemLostForAClosedLoan() {
+
+    assertThat(accountsForLoan, hasItem(allOf(
+      hasJsonPath("feeFineType", "Lost item processing fee"),
+      hasJsonPath("amount", expectedProcessingFee))));
+  }
+
+  @Test
+  public void shouldNotAssignFeesWhenUnknownPolicy() {
+    final InventoryItemResource item = itemsFixture.basedUponNod();
+    final IndividualResource loan = loansFixture.checkOutByBarcode(item, usersFixture.charlotte());
+
+    final JsonObject loanWithoutLostPolicy = loansStorageClient.getById(loan.getId())
+      .getJson().copy();
+    loanWithoutLostPolicy.remove("lostItemPolicyId");
+
+    loansStorageClient.replace(loan.getId(), loanWithoutLostPolicy);
+    assertThat(loansStorageClient.getById(loan.getId()).getJson().getString("lostItemPolicyId"),
+      nullValue());
+
+    declareLostFixtures.declareItemLost(new DeclareItemLostRequestBuilder()
+      .forLoanId(loan.getId()));
+
+    assertThat(getAccountsForLoan(loan.getId()), hasSize(0));
+  }
+
+  @Test
+  public void shouldNotAssignFeesWhenNoOwnerConfigured() {
+    feeFineOwnerFixture.delete(feeFineOwnerFixture.cd1Owner());
+
+    final InventoryItemResource item = itemsFixture.basedUponNod();
+    final IndividualResource loan = loansFixture.checkOutByBarcode(item, usersFixture.charlotte());
+
+    declareLostFixtures.declareItemLost(new DeclareItemLostRequestBuilder()
+      .forLoanId(loan.getId()));
+
+    assertThat(getAccountsForLoan(loan.getId()), hasSize(0));
+  }
+
+  @Test
+  public void cannotDeclareItemLostWhenNoAutomatedLostItemFee() {
+    feeFineTypeFixture.delete(feeFineTypeFixture.lostItemFee());
+
+    final InventoryItemResource item = itemsFixture.basedUponNod();
+    final IndividualResource loan = loansFixture.checkOutByBarcode(item, usersFixture.charlotte());
+
+    final Response response = declareLostFixtures.attemptDeclareItemLost(
+      new DeclareItemLostRequestBuilder()
+        .forLoanId(loan.getId()));
+
+    assertThat(response.getJson(), hasErrorWith(allOf(
+      hasMessage("Expected automated fee of type Lost item fee"),
+      hasParameter("feeFineType", "Lost item fee"))));
+  }
+
+  @Test
+  public void cannotDeclareItemLostWhenNoAutomatedLostItemProcessingFee() {
+    feeFineTypeFixture.delete(feeFineTypeFixture.lostItemProcessingFee());
+
+    final InventoryItemResource item = itemsFixture.basedUponNod();
+    final IndividualResource loan = loansFixture.checkOutByBarcode(item, usersFixture.charlotte());
+
+    final Response response = declareLostFixtures.attemptDeclareItemLost(
+      new DeclareItemLostRequestBuilder()
+        .forLoanId(loan.getId()));
+
+    assertThat(response.getJson(), hasErrorWith(allOf(
+      hasMessage("Expected automated fee of type Lost item processing fee"),
+      hasParameter("feeFineType", "Lost item processing fee"))));
+  }
+
+  @Test
+  @Parameters( {
+    "0",
+    "10.00"
+  })
+  public void shouldNotAssignProcessingFeeIfDisabled(double processingFee) {
```

Comment:
Is this when no fees should be charged due to the policy or only the item processing fee?

Reply:
Only the processing fee should not be charged, but item fee should be charged in case amount > 0 - this is covered by `api.loans.DeclareLostAPITests#canAssignItemFeeOnly`

Reply:
Does `withChargeAmountItemPatron` mean `charge lost item fees to patron?` ?

Reply:
No, it means whether the **processing fee** have to be charged or not. The item fee is not controlled by this flag.

Reply:
Ok, so `chargeAmountItemPatron` means `charge the lost item processing fee`?

Reply:
Yes, correct.

Reply:
I think the builder method should reflect the intent of the setting and hide the rather vague property name from the tests.

For example `chargeProcessingFee` and `doNotChargeProcessingFee`.

What do you think?

Does the production code also use this naming or is it hidden behind the policy / repository?

Reply:
Production code use the same name as it is defined in feefines. Should I rename it as well?

Reply:
> Production code use the same name as it is defined in feefines. Should I rename it as well?

Yes please, I think the code should describe the intent of the setting and hide the (unclear to me at least) name

Reply:
Add `chargeProcessingFee` and `doNotChargeProcessingFee` methods for test builder and `shouldChargeProcessingFee` to `LostItemPolicy`.